README.md

```diff
@@ -36,6 +62,25 @@ https://abdo9826.github.io/protofolio/
 - Twitter: [@Abdelrahman Mohamed](https://twitter.com/abodyalex1)
 - LinkedIn: [Abdelruhman Mihamed](https://www.linkedin.com/in/abdelruhman-mihamed-a42667179/)
 
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to code reviewers and to everyone who reviewed the project and made suggestions.
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
- [ ] Kindly copy the [MIT license](https://github.com/microverseinc/readme-template/blob/master/MIT.md) provided in [Microverse's readme template](https://github.com/microverseinc/readme-template), as suggested by the previous reviewer.